src/ua/hillel/Animals.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Scanner;
+
+public class Animals {
+    int add_animals;
+
+private String name;
```

Comment:
Please apply proper formatting to all classes of this homework. Please never forget to do it before pushing your code on review!